template/golang-http/go.mod

```diff
@@ -2,4 +2,8 @@ module handler
 
 go 1.18
 
-require github.com/openfaas/templates-sdk/go-http v0.0.0-20220408082716-5981c545cb03
+replace handler/function => ./function
+
+require (
+	github.com/openfaas/templates-sdk v0.0.0-20200723092016-0ebf61253625
```

Comment:
i think we want to use the current sha here instead of reverting this value, this keeps the template-sdk up-to-date

Reply:
yea makes perfect sense, only thing I was not sure about is the difference between 2020 and 2022 in this repository in terms of reliability. Will do the change